src/main/cpp/subsystems/Climber.cpp

```diff
@@ -193,5 +194,9 @@ bool Climber::IsClimberUp() {
 }
 
 int Climber::GetLiftMotorEncoderValue() {
-    return mLiftMotorEncoder.GetPosition();
+    return mLiftMotorEncoder.GetPosition() - mEncoderOffset;
+}
+
+void Climber::ResetEncoder() {
+    mEncoderOffset = GetLiftMotorEncoderValue();
```

Comment:
You can't call `GetLiftMotorEncoderValue()`. You have to access the encoder directly.

`GetLiftMotorEncoderValue` has the offset!